### DISCUSSION TITLE
Getting Unauthenticated when update to Laravel 8, Passport 10

### DISCUSSION THREAD

**Discussion by trancongminh132** [laravel/framework#36320] [Q&A]
Hello, currently, I upgraded to Laravel 8, Passport 10.
When I try to call oauth/token, it return bearer_token ok,
But when I use that bearer token that to call some api, it always return 
"Unauthenticated"
Could someone help me?
Thank you!

**Comment by KateKoltsova** [comment]
you should write more information about your problem, for example, are you sure that you are passing the token correctly?

**Reply by henzeb** [reply]
This is old. Needs to be closed.
